src/org/zaproxy/zap/extension/codedx/CodeDxExtension.java

```diff
@@ -1,30 +1,38 @@
 /*
- * Zed Attack Proxy (ZAP) and its related class files.
- * 
- * ZAP is an HTTP/HTTPS proxy for assessing web application security.
- * 
- * Licensed under the Apache License, Version 2.0 (the "License"); 
- * you may not use this file except in compliance with the License. 
- * You may obtain a copy of the License at 
- * 
- *   http://www.apache.org/licenses/LICENSE-2.0 
- *   
- * Unless required by applicable law or agreed to in writing, software 
- * distributed under the License is distributed on an "AS IS" BASIS, 
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. 
- * See the License for the specific language governing permissions and 
- * limitations under the License. 
+ * Copyright (C) 2016 Code Dx, Inc. - http://www.codedx.com

```

Comment:
We usually expect new code to be Copyright the ZAP Development team. We'll need to discuss what alternatives we will allow and under what circumstances...